src/quic_gateway.rs

```diff
@@ -1094,6 +1771,24 @@ impl GatewayIdentityUpdater {
     }
 }
 
+#[async_trait::async_trait]
+impl JetIdentitySyncMember for GatewayIdentityUpdater {
+    async fn pause_for_identity_update(
+        &self,
+        new_identity: Keypair,
+        barrier: Arc<tokio::sync::Barrier>,
+    ) {
+        let cmd = MultiStepIdentitySynchronizationCommand {
+            new_identity,
+            barrier,
+        };
+        self.cnc_tx
+            .send(GatewayCommand::MultiStepIdentitySynchronization(cmd))
+            .await
+            .expect("disconnected");
+    }
```

Comment:
I reviewing the different pausable members this one is async but another is sync.